dev/import-beats/spelling.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"strings"
+)
+
+var spellingReplacer = strings.NewReplacer(
+	"mysql", "MySQL", "Mysql", "MySQL")
```

Comment:
this doesn't belong here. I think we need to fix it in the source data, which is a perfectly fine thing to do